integration/virtual_guest_metadata/virtual_guest_metadata_test.go

```diff
@@ -31,7 +31,7 @@ var _ = Describe("SoftLayer Virtual Guest Lifecycle", func() {
 		testhelpers.POLLING_INTERVAL = 10 * time.Second
 	})
 
-	Context("SoftLayer_VirtualGuestService#setUserMetadata and SoftLayer_VirtualGuestService#configureMetadataDisk", func() {
+	XContext("SoftLayer_VirtualGuestService#setUserMetadata and SoftLayer_VirtualGuestService#configureMetadataDisk", func() {
```

Comment:
Since we are committing this with the X let's add a comment (brief) explaining and also reference to tracker story and github issue to address in future. Thanks